testdata/p4tc_samples_outputs/direct_counter_example.template

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -x
+
+set -e
+
+TC="tc"
+$TC p4template create pipeline/direct_counter_example numtables 1
+
+$TC p4template create action/direct_counter_example/ingress/send_nh actid 1 \
+	param port_id type dev \
+	param dmac type macaddr \
+	param smac type macaddr
+$TC p4template update action/direct_counter_example/ingress/send_nh state active
+
+$TC p4template create action/direct_counter_example/ingress/drop actid 2
+$TC p4template update action/direct_counter_example/ingress/drop state active
+
+$TC p4template create table/direct_counter_example/ingress/nh_table \
+	tblid 1 \
+	type exact \
+	keysz 32 nummasks 8 permissions 0x18a6 tentries 2048 \
+	pna_direct_counter Counter/ingress.global_counter \
+	table_acts act name direct_counter_example/ingress/send_nh \
+	act name direct_counter_example/ingress/drop
+$TC p4template update table/direct_counter_example/ingress/nh_table default_miss_action permissions 0x1024 action direct_counter_example/ingress/drop
+
+$TC p4template create extern//root/DirectCounter extid 1 numinstances 1 tc_acl 0x19b6 has_exec_method 
+
+$TC p4template create extern_inst/direct_counter_example/DirectCounter/ingress.global_counter instid 1 \
```

Comment:
Direct Counter template and instance declaration should come before the table creation because the extern instance needs to exist before you associate it to a table